src/monocle_apptrace/constants.py

```diff
@@ -0,0 +1,7 @@
+AZURE_ML_ENDPOINT_ENV_NAME = "AZUREML_ENTRY_SCRIPT"
```

Comment:
can we leverage this association with a dictionary /tuple 

az_env_properties ={
'FUNCTIONS_WORKER_RUNTIME':'azure_function',
'WEBSITE_SITE_NAME':'azure_app_service'
...
}